wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -1686,11 +1744,13 @@ instance BuildableSafeGen NodeInfo where
         %" blockchainHeight="%build
         %" localBlockchainHeight="%build
         %" localTimeDifference="%build
+        %" subscriptionStatus="%listJson
         %" }")
         nfoSyncProgress
         nfoBlockchainHeight
         nfoLocalBlockchainHeight
         nfoLocalTimeInformation
+        (toList nfoSubscriptionStatus)
```

Comment:
If `toList` is from `Data.Foldable`, then this will do `Map k v -> [v]`. We likely want `Map.toList :: Map k v => [(k, v)]` ?

Reply:
Ho :o! That I didn't know. Will use a qualified import here too :+1: